src/components/Rockets.js

```diff
@@ -0,0 +1,28 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { cancelReservation, reserveRocket } from '../redux/rockets/rockets';
+import '../styles/rockets.css';
+
+const Rockets = () => {
+  const rockets = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <ul className="rocket-ul">
+        {rockets.map((r) => (
+          <li key={r.id} className="rocket-li">
+            <img className="rocket-img" src={r.flickr_images} alt={r.id} />
+            <div className="rocket-txt">
+              <h2>{r.name}</h2>
+              {r.reserved === true ? <span className="reserveBtn">&bull; Reserved</span> : ''}
+              <p>{r.description}</p>
```

Comment:
[OPTIONAL] The colors you have used make it difficult to see the text `Reserved` when a reservation is made as shown below since it appears as just a normal text. Kindly update it to match the actual design so that it can stand out. :wink: 

![Screenshot from 2023-02-03 11-59-36](https://user-images.githubusercontent.com/65233860/216598644-74816c16-3c5c-4f7b-bd62-0f5b61d77e37.png)